book/intro.md

```diff
@@ -1,11 +1,89 @@
-# Welcome to your Jupyter Book
+# 🥳 Welcome to the EGU22 PyGMT short course
 
-This is a small sample book to give you a feel for how book content is
-structured.
-It shows off a few of the major file types, as well as some sample content.
-It does not go in-depth into any particular topic - check out [the Jupyter Book documentation](https://jupyterbook.org) for more information.
+This Jupyter book contains [PyGMT](https://www.pygmt.org/v0.6.1) tutorials
+for producing maps and doing geospatial data processing 🌍
 
-Check out the content pages bundled with this sample book to see more.
+- [Anatomy of a PyGMT figure](first-figure) by [Leonardo Uieda](https://orcid.org/0000-0001-6123-9515)
+- [Integration with the scientific Python ecosystem 🐍](ecosystem) by [Max Jones](https://orcid.org/0000-0003-0180-8928)
```

Comment:
Max, let me know if you prefer to use your new name here. I can also update your name on the main README.md. There should be a way to change your name at https://administrator.copernicus.org/personal_data, though I'm not sure if that will show up immediately on the abstract page.

Reply:
Thanks for checking in. This is good here and it would be nice to update the readme (let me know if you want me to submit a PR). I just updated my EGU profile.

Reply:
Sure, you can start a PR to update the main README.md. Maybe a good idea to update the 'Schedule' table too.